source/extensions/tracers/opentelemetry/grpc_trace_exporter.h

```diff
@@ -6,6 +6,7 @@
 #include "source/common/grpc/typed_async_client.h"
 
 #include "opentelemetry/proto/collector/trace/v1/trace_service.pb.h"
+#include "trace_exporter.h"
```

Comment:
please use full-path

Reply:
Done in: [2a5d544](https://github.com/envoyproxy/envoy/pull/29207/commits/2a5d5448406e99fbfaecda651e0d176eda96383a)